src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -95,6 +95,40 @@
                  version))))
     version))
 
+(defn level->int
+  "Returns an integer which represents the given status level.
+   The ordering of levels is :critical < :info < :debug."
+  [level]
+  (case level
+    :critical 0
+    :info 1
+    :debug 2))
+
+(defn apply-to-int-levels
```

Comment:
That was my original thought as well, as that's how we intended it to be used, but what it actually does it more general than just a comparison, so that name doesn't seem quite right to me.